fala/locale/cy/LC_MESSAGES/django.po

```diff
@@ -31,7 +31,7 @@ msgstr "Rhowch enw eich sefydliad"
 msgid "Enter a valid postcode"
 msgstr "Rhowch god post dilys"
 
-#: fala/common/base_form_components.py:30 fala/templates/adviser/search.html:45
+#: fala/common/base_form_components.py:30 fala/templates/adviser/category_search.html fala/templates/adviser/search.html:45
```

Comment:
@willc-work just checking that you did the folloing:

- ran `python manage.py makemessages -l cy -i locale -i venv` to get an updated `django.po` file
- then added translations into `msgstr`
- then ran `python manage.py compilemessages -l cy` to see the changes happen locally?

Reply:
oops, they should be visible now